src/Datadog.Trace/SpanContextPropagator.cs

```diff
@@ -86,6 +87,23 @@ public SpanContext Extract<T>(T headers)
             return new SpanContext(traceId, parentId, samplingPriority, null, origin);
         }
 
+        public IEnumerable<KeyValuePair<string, string>> ExtractHeaderTags<T>(T headers, IEnumerable<KeyValuePair<string, string>> headerTags)
+            where T : IHeadersCollection
+        {
+            Dictionary<string, string> tagsFromHeaders = new Dictionary<string, string>();
```

Comment:
I think even @macrogreg would use `var` here 😅
```suggestion
            var tagsFromHeaders = new Dictionary<string, string>();
```